frameworks/remix/app/services/auth.server.ts

```diff
@@ -0,0 +1,21 @@
+import { Authenticator } from "remix-auth";
+import { KeycloakStrategy } from "remix-keycloak";
+import { sessionStorage } from "~/services/session.server";
+
+export const authenticator = new Authenticator(sessionStorage);
+
+const keycloakStrategy = new KeycloakStrategy(
+  {
```

Comment:
@rishi-raj-jain let's pull out this object and make it a named item independent of this, this will allow for referencing it easier in the write up